### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jnweiger**
Added a few lines how to use ownbuild with linux.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,15 @@ This repository contains a helper script to mak it easier to setup a ownCloud bu
 - `(py.exe|python3) ownbuild.py --branch 2.10 --target windows-msvc2019_64-cl`
 ### Query available targets
 - `(py.exe|python3) ownbuild.py --branch 2.10 --target help`
+
+
+## On Linux
+
```
Comment (jnweiger) [inline README.md:39]: Prerequisites gcc, git, cmake, ... etc 

Note: ownbuild / craft uses cached binary packages from https://download.owncloud.com/desktop/craft/cache/
to speed up the compilation.

Comment (TheOneRing) [reply README.md:39]: ownbuild will provides cmake so we probably only need the compiler and git and well python

**Review by mmattel — APPROVED** [review]
We have tested this in an online session with @jnweiger, LGTM, approving

File: README.md [hunk 2]
```diff
@@ -33,3 +33,15 @@ This repository contains a helper script to mak it easier to setup a ownCloud bu
...
+### With owncloud dependencies:
+
+- `python3 ./ownbuild.py --branch master --target linux-64-gcc owncloud-client`
+- When built, start the client like this: `./master/linux-64-gcc/bin/owncloud -s`
+
+### With system dependencies:
```
Comment (fmoc) [inline README.md]: What's the point of "with system dependencies"? The entire point of ownbuild is to use craft to build binaries, with the majority of system dependencies coming from craft. The majority is maintained by KDE, not ownCloud. All you need to document are a few system dependencies. These are probably pretty much equal to what's installed in the `kdeorg/ubuntu-1804-craft`Docker image, so we can just copy the list from there.

Comment (jnweiger) [reply README.md]: TODO: find the relevant parts from kdeorg/ubuntu-1804-craft docker image.

Comment (TheOneRing) [reply README.md]: As the cache is used those system dev packages are not needed.

**Pull request merged**